examples/ima-video.amp.html

```diff
@@ -0,0 +1,33 @@
+<!doctype html>
```

Comment:
they would need dev console. https://www.ampproject.org/docs/reference/experimental has good instructions. Maybe add a paragraph to the example mentioning it is experimental and point to the documentation?

(while you are at it, could you add `layout=responsive` to `amp-ima-video` in the example?)